### ISSUE TITLE
App no longer available in Play?

### ISSUE THREAD

**Issue by LunaticoCR** [TrentPierce/DontPause#7]
Hi,

I've just noticed this app is no longer in the Play Store, is this on purpose?

Thanks!

**Comment by TrentPierce** [comment]
Here's the apk if you want it. 


Sent via the Samsung Galaxy S9+, an AT&T 4G LTE smartphone
-------- Original message --------From: Daniel <notifications@github.com> Date: 8/31/18  9:59 AM  (GMT-06:00) To: TrentPierce/DontPause <DontPause@noreply.github.com> Cc: Subscribed <subscribed@noreply.github.com> Subject: [TrentPierce/DontPause] App no longer available in Play? (#7) 
Hi,
I've just noticed this app is no longer in the Play Store, is this on purpose?
Thanks!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/TrentPierce/DontPause","title":"TrentPierce/DontPause","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/TrentPierce/DontPause"}},"updates":{"snippets":[{"icon":"DESCRIPTION","message":"App no longer available in Play? (#7)"}],"action":{"name":"View Issue","url":"https://github.com/TrentPierce/DontPause/issues/7"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/TrentPierce/DontPause/issues/7",
"url": "https://github.com/TrentPierce/DontPause/issues/7",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "App no longer available in Play? (#7)",
"sections": [
{
"text": "",
"activityTitle": "**Daniel**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@LunaticoCR",
"facts": [
{
"name": "Repository: ",
"value": "TrentPierce/DontPause"
},
{
"name": "Issue #: ",
"value": 7
}
]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"TrentPierce/DontPause\",\n\"issueId\": 7,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"TrentPierce/DontPause\",\n\"issueId\": 7\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/TrentPierce/DontPause/issues/7"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 374700671\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by LunaticoCR** [comment]
That'd be great if you could share the apk! Did you mean to attach it somehow? Maybe a link? Thanks!

**Comment by LunaticoCR** [comment]
Hey, any chance you could share that apk?

**Comment by hartmark** [comment]
I realized also it was gone from play store when I got my new phone. 

@TrentPierce any plans on putting it back up?

I'll try build a new APK from the source and if the project is dead I can fork it and put it back on play store

**Comment by TrentPierce** [comment]
I'll get it back up there soon since I still have the key file. Do you need
the apk for your new phone?

**Comment by hartmark** [comment]
Cool! Yeah I'd love the APK for my new phone. You can perhaps put it up here on the release tab?

**Comment by TrentPierce** [comment]
Yeah, I'll do that. I don't know why I didn't think to do that sooner

**Comment by LunaticoCR** [comment]
Great! Thanks for uploading it!

**Comment by hartmark** [comment]
Awesome! It installed perfectly. Thanks!

**Comment by sagikerius** [comment]
Is there a way I can get the app?? I just had to factory reset my phone and my app is gone.... I loved this!!
